src/modules/services.js

```diff
@@ -155,7 +155,7 @@ module.exports = {
     myDb.pragma('journal_mode = WAL');
 
     myDb.pragma('SYNCHRONOUS = 1;');
-    myDb.pragma('LOCKING_MODE = EXCLUSIVE;');
+    // myDb.pragma('LOCKING_MODE = EXCLUSIVE;');
```

Comment:
why?

Reply:
I specified it on the commit message bit it got lost in the log :) It's basically cos sqlite3 is for some reason not allowing multiple connections to the database, one from the fork and from the main app. Probably worth locking the queries instead of the connection to the db. Happy to change to for a different approach if you have any ideas